src/components/Footer.vue

```diff
@@ -1,80 +1,241 @@
+              <a
+                href="https://www.facebook.com/nasaspaceappscolombo/"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Facebook</a
+              >
+            </li>
+            <li>
+              <i class="sm-icon fab fa-instagram"></i>
+              <a
+                href="https://www.instagram.com/nasa_spaceapps_colombo/?r=nametag"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
+            </li>
+            <li>
+              <i class="sm-icon fab fa-youtube"></i>
+              <a
+                href="https://www.youtube.com/channel/UCTLG4OxFsG0a-TrPlLGwtbw?view_as=subscriber"
+                target="_blank"
+                rel="noopener noreferrer"
+                >YouTube</a
+              >
+            </li>
+            <li>
+              <i class="sm-icon fab fa-twitter"></i>
+              <a
+                href="https://twitter.com/SpaceAppsLK"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Twitter</a
+              >
+            </li>
+            <li>
+              <i class="sm-icon fab fa-linkedin"></i>
+              <a
+                href="https://www.linkedin.com/company/nasa-space-apps-colombo"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+            </li>
+          </ul>
+        </div>
+        <div class="column is-3 has-text-left footer-col">
+          <h1 class="title is-4 text-pink">Contact Us</h1>
+          <ul class="contact-info">
+            <li>
+              <i class="sm-icon fas fa-phone"></i>
+              <a
+                href="tel:+94712068707"
+                target="_blank"
+                rel="noopener noreferrer"
+                >+94 71 206 8707</a
+              >
+            </li>
+            <li>
+              <i class="sm-icon fas fa-envelope"></i>
+              <a
+                href="mailto:info@spaceappscolombo.org"
+                target="_blank"
+                rel="noopener noreferrer"
+                >info@spaceappscolombo.org</a
+              >
+            </li>
+            <li>SEDS Sri Lanka,</li>
+            <li>Colombo</li>
+          </ul>
         </div>
+      </div>
     </div>
-    <p class="has-text-centered mt-3">Copyright © 2022 SEDS Sri Lanka. All Rights Reserved</p>
+    <p class="has-text-centered mt-3">
+      Copyright © 2024 SEDS Sri Lanka. All Rights Reserved
+    </p>
   </footer>
 </template>
 
 <script>
 export default {
-    name: 'Footer'
+  name: "Footer",
 };
 </script>
 
 <style scoped>
 @import url("../assets/css/style.css");
-footer{
-    background-color: #040127 !important;
+
+footer {
+  background-color: #020202 !important; /* Darker background color for contrast */
+  color: #fff; /* Ensure text is visible on dark background */
```

Comment:
We dont need these comments here. The code is self-explanatory

Reply:
Check other places too. Only add comments when the code itself doesn't convey its usage.